src/main/kotlin/de/cyface/collector/model/Uploadable.kt

```diff
@@ -109,7 +109,21 @@ interface Uploadable {
         val properties = JsonObject()
         val geometry = JsonObject()
 
+        // DeviceMetaData
+        properties.put(FormAttributes.OS_VERSION.value, deviceMetaData.operatingSystemVersion)
+        properties.put(FormAttributes.DEVICE_TYPE.value, deviceMetaData.deviceType)
+
+        // ApplicationMetaData
+        properties.put(FormAttributes.APPLICATION_VERSION.value, applicationMetaData.applicationVersion)
+        properties.put(FormAttributes.FORMAT_VERSION.value, applicationMetaData.formatVersion)
+
+        // MeasurementMetaData
+        properties.put(FormAttributes.LENGTH.value, measurementMetaData.length)
+        properties.put(FormAttributes.LOCATION_COUNT.value, measurementMetaData.locationCount)
         if (measurementMetaData.startLocation != null && measurementMetaData.endLocation != null) {
+            properties.put(FormAttributes.START_LOCATION_TS.value, measurementMetaData.startLocation.timestamp)
+            properties.put(FormAttributes.END_LOCATION_TS.value, measurementMetaData.endLocation.timestamp)
```

Comment:
We almost dropped this information by only storing start/end location's geometry.

This information is required as the web-app also shows non-deserialized but already uploaded measurements, and the measurement date filter uses the `start` and `end` timestamp of the `metadata`.